data/src/main/java/io/gnosis/data/backend/dto/TransactionDto.kt

```diff
@@ -57,5 +57,11 @@ sealed class ParamDto {
 
 enum class Operation(val id: Int) {
```

Comment:
I would not assume any knowledge about UI layer here. Better to have an extension function in the fragment

Reply:
Done in 4ed1267d9732e101ba665de5cdec3515f1817a7a